packages/platform-core/src/cwd_package_json_loader.ts

```diff
@@ -0,0 +1,20 @@
+import * as path from 'path';
+import { PackageJsonReader } from '.';
+
+/**
+ * Loads the contents of package.json from process.cwd().
+ *
+ * Throws if no package.json is present
+ */
+export class CwdPackageJsonLoader {
+  /**
+   * Returns the contents of the package.json file in process.cwd()
+   *
+   * If no package.json file exists, or the content does not pass validation, an error is thrown
+   */
+  read() {
```

Comment:
```suggestion
  read(): PackageJson {
```

this might help

Reply:
updated

Reply:
I might have forgotten that this needs `Promise<PackageJson>` if this is async.

Reply:
nvm I can see the comment in code.